packages/muelu/src/Misc/MueLu_CoordinatesTransferFactory_def.hpp

```diff
@@ -126,7 +126,7 @@ namespace MueLu {
   void CoordinatesTransferFactory<Scalar, LocalOrdinal, GlobalOrdinal, Node>::Build(Level & fineLevel, Level &coarseLevel) const {
     FactoryMonitor m(*this, "Build", coarseLevel);
 
-    using xdMV = Xpetra::MultiVector<typename Teuchos::ScalarTraits<Scalar>::coordinateType,LO,GO,NO>;
```

Comment:
I believe we assume that coordinates are always `double`.  I guess this doesn't play well with Stokhos?

Reply:
After changing from `coordinateType` to `magnitudeType` it compiles on my system. I'm not sure I understand why. Maybe a there is a difference in what these map to for Sacado?

Reply:
I recall the coordinate type was introduced by Kim to fix something with Stokhos. I am not sure what the issue was with magnitude type, I suspect though that it is not necessarily a scalar for Sacado/Stokhos...